.github/workflows/tests.yml

```
@@ -20,13 +20,14 @@ jobs:
         rust: ["stable", "beta"]
         deny_warnings: ['', 1]
 
-        # For nightly rust, run the check-exercises.sh script with DENYWARNINGS=1
+        # TODO: I don't know how to make it run only the check_exercises.sh script
+        # For nightly rust, run the job with DENYWARNINGS=1
         include:
           - rust: nightly
             deny_warnings: 1
     
     # Allow failures on nightly rust, or beta if DENYWARNINGS is enabled
-    continue-on-error: ${{ matrix.rust == "nightly" || (matrix.rust == "beta" && DENYWARNINGS == 1)}}
+    continue-on-error: ${{ matrix.rust }} == "nightly" || ( ${{ matrix.rust }} == "beta" && DENYWARNINGS == 1 )
```

Having a bit of trouble expressing this condition in a way that pleases GitHub Actions. I get this error message:

```
Error when evaluating 'continue-on-error' for job 'run-all-tests'. (Line: 30, Col: 24): Unexpected value 'stable == "nightly" || ( stable == "beta" && DENYWARNINGS == 1 )'
```

Does anyone know what the correct syntax/placement here is? Hard to test it with my setup because I have to make an entire commit + push to actually run it.

---

I think you're missing quotation marks around `${{ matrix.rust }}`

---

I don't know for sure, but I suspect that the entire expression has to go inside a `${{ ... }}` group; [docs](https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions). 

At a guess, therefore: 
```suggestion
    continue-on-error: ${{ matrix.rust == 'nightly' || ( matrix.rust == 'beta' && matrix.deny_warnings == '1'  ) }}
```